etc/input.conf

```diff
@@ -65,6 +65,8 @@
 #ZOOMIN    add video-zoom   0.1         # zoom in
 #Alt+-     add video-zoom  -0.1         # zoom out
 #ZOOMOUT   add video-zoom  -0.1         # zoom out
+#Ctrl+WHEEL_UP   add video-zoom   0.1   # zoom in
+#Ctrl+WHEEL_DOWN add video-zoom  -0.1   # zoom out
```

Comment:
You could remove one space before the amounts, the previous bindings have it to align with add `video-pan-*  0.1` but these aren't aligned with those anyway.